<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgMemClass</class>
 <widget class="QWidget" name="WdgMemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WdgMem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WdgMemClass {
		background-color: #2a2a2a;
        border: 0px;
}

QWidget {
		background-color: transparent;
        border: 0px;
}

QLabel {
		background-color: transparent;
        color: white;
}

QLineEdit{
	background-color: #2a2a2a;
	border: 1px solid;
	border-color: #909090;
	color: white;
	selection-background-color: rgb(70, 150, 70);
}
/***********************************************/
QTabWidget::pane { 
	top: 1px;
	border: 1px solid;
	border-color: black;
	background-color: #2f2f2f;
}

QTabWidget::tab-bar {
	top: 1px;
	left: 1px;
}

QTabWidget::QWidget {
	background-color: #009900;
}

QTabBar::tab {
	background-color: #2a2a2a;
	border: 1px solid;
	border-color: black;
	color: white;
	min-width: 16px;
	padding: 3px;
}

QTabBar::tab:selected {
	background-color: #2f2f2f;
}

QTabBar::tab:hover {
	background-color: #363636;
}

QTabBar::tab:!selected {
	margin-left: 0px;
}
QTabBar::tab:selected {
	margin-bottom: 0px;
	margin-top: 0px;
	border-right: 0px;
}
QTabBar::tab:first:selected {
	margin-bottom: 0; 
	border-bottom: 0px solid;
	border-bottom-color:rgb(150, 150, 150);
}
QTabBar::tab:first{
	margin-bottom: 0; 
	border-bottom: 0px solid;
	border-bottom-color:rgb(150, 150, 150);
}
QTabBar::tab:last:selected {
	margin-top: 0;
}
QTabBar::tab:only-one {
	margin: 0;
}
/***************************************************************/
QPushButton {
border: 1px solid #000000;
border-radius: 0px;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #616161, stop:0.3 #595959, stop:0.35 #4C4C4C, stop: 1 #424242);
color: #EBEBEB;
}
QPushButton:hover {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #c2c2c2, stop:0.5 #959595, stop:0.65 #959595, stop: 1 #707070);
}
QPushButton:disabled {
color: #787878;
}
QPushButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #325C32, stop:0.45 #366436, stop:0.5 #366436, stop: 1 #4A944A);
}
QPushButton:flat {
border: none;
}
QPushButton::checked {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #325C32, stop:0.45 #366436, stop:0.55 #366436, stop: 1 #4e9d4e);
}
/***********************************************/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>  Memory  </string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAdd">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbDel">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Del</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbComment">
            <property name="text">
             <string>Comment:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leComment">
            <property name="toolTip">
             <string>tooltip Comment</string>
            </property>
            <property name="statusTip">
             <string>st Input comment</string>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbMemory">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>M</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twMem">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView {
	background: transparent;
	color: white;
    border: 0px;
    gridline-color: #808080;
	selection-background-color: #424242;
	alternate-background-color: transparent;
 }

QHeaderView::section {
	color: white;
	border-radius: 1px;
    border: 1px solid #808080;
	background-color: #353535;
    padding-left: 4px;
 }

QHeaderView::down-arrow {
	image: url(:images/arrows/arrow-down.png);
 }

QHeaderView::up-arrow {
	image: url(:images/arrows/arrow-up.png);
 }

QScrollBar:vertical { 
border: 1px solid black; 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(60, 60, 60, 255), stop:1 rgba(50, 50, 50, 255)); 
width: 13px;
margin: 12px 3px 12px 3px; 
} 

QScrollBar::handle:vertical {
	background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(60, 60, 60, 255), stop:1 rgba(137, 137, 137, 255));
	border: 1px solid;
	border-color: rgb(150, 150, 150);
	border-radius: 1px;
	min-width: 18px;
	margin: 3px -2px 3px -2px; 
}

QScrollBar::add-line:vertical 
{
	background: transparent;
	height: 11px;
	width: 11px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
	image: url(:images/arrows/arrow-down.png);
  }

 QScrollBar::sub-line:vertical 
{
	background: transparent;
	height: 11px;
	width: 11px;
	subcontrol-position: top;
	subcontrol-origin: margin;
	image: url(:images/arrows/arrow-up.png);
  }

QScrollBar::up-arrow:vertical
{
      width: 11px;
      height: 11px;
  }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::CustomDashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>67</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Comment</string>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Frequency</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mode</string>
          </property>
          <property name="textAlignment">
           <set>AlignLeft|AlignVCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tapIqReq">
      <attribute name="title">
       <string>  IQ Recorder  </string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
